apps/tup-cms/src/taccsite_cms/settings_custom.py

```diff
@@ -192,6 +192,8 @@
 TACC_BLOG_CUSTOM_MEDIA_POST_CATEGORY = 'multimedia'
 TACC_BLOG_SHOW_ABSTRACT_TAG = 'external'
 
```

Comment:
**Where to Put Setting?**
Always a work in progress. I think it could fit in a **new section just above** the "PLUGIN SETTINGS" section, or as **a miscellaneous item within** the "PLUGIN SETTINGS" section, because it is after "TACC" settings (from Core-CMS)[^1] and near/in miscellaneous plugin settings.

If the new section route sounds good, here are some ideas:
```suggestion

########################
# TACC: STAFF PROFILE
########################
```

This, and move to just above the "PLUGIN SETTINGS" section.
```suggestion

########################
# TACC PLUGIN SETTINGS
########################
```

[^1]: The misleading "TACC: " prefix will be revisited with or after [TACC/Core-CMS#792](https://github.com/TACC/Core-CMS/pull/792) where I replace the `TACC_` prefix.

Reply:
Fixed.